packages/tools/buildServerDocker.ts

```diff
@@ -18,6 +19,9 @@ async function main() {
 	const tagName = argv.tagName;
 	const isPreRelease = getIsPreRelease(tagName);
 	const imageVersion = getVersionFromTag(tagName, isPreRelease);
+	const buildDate = moment(new Date().getTime()).format('YYYY-MM-DDTHH:mm:ssZ');
+	const revision = await execCommand2('git rev-parse --short HEAD');
```

Comment:
One problem we had with running git commands is that people sometimes run this outside of git repos, and then the whole script fails. So it it possible to set the revision to an empty string if the git command fails?

Also I'm not certain how it will work on CI. I think on Travis it was building from a headless branch, and not sure about GitHub Actions, but I guess we'll try and see.

Reply:
For info, this is the script I mentioned: https://github.com/laurent22/joplin/blob/ed7face070c2bf7b8fe1e3f351b460e4f3ba1015/packages/app-desktop/tools/compile-package-info.js#L26

Reply:
Good hint! I'll catch the error now and set a empty string. 

>Would that be a problem if that variable is set to an empty string in the Dockerfile?

No, because this is only a label for displaying information and has no funktion for the image. 

![grafik](https://user-images.githubusercontent.com/24863925/124645874-3f181100-de94-11eb-8e8d-4a4cc3aed6e9.png)